package.json

```diff
@@ -2,7 +2,7 @@
   "name": "packwerk-vscode",
```

Comment:
I believe this string is used as part of the extension unique identifier, so changing the `displayName` should be fine.